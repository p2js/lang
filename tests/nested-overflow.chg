fn main() {
  let x = bitcast(0);
  x = bitcast(1);
}
