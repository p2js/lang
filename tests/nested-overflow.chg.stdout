expr @ 2:11 2:18 => extern "intrinsic" fn bitcast(...)
expr @ 2:19 2:20 => i32
expr @ 2:11 2:21 => ?1t
expr @ 3:7 3:14 => extern "intrinsic" fn bitcast(...)
expr @ 3:15 3:16 => i32
expr @ 3:7 3:17 => ?3t
expr @ 3:3 3:4 => ?1t
expr @ 3:3 3:17 => ()
expr @ 1:11 5:1 => ()

